modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -6938,7 +6938,10 @@ public VirtualHostModel newVirtualHostModel() {
 				BenchmarksPropsValues.ADMIN_INSTANCE_VIRTUAL_HOST_NAME);
 		}
 		else {
-			virtualHostModel.setHostname(_webId);
+			virtualHostModel.setHostname(
+				StringBundler.concat(
+					BenchmarksPropsValues.VIRTUAL_HOST_NAME_PREFIX,
+					_webId.substring(7, 8), ".com"));
```

Comment:
@slnn `subString(7, 8)` won't work if there are 10 or more companies.

Reply:
Oh, sorry, will fix it

Reply:
Oh, sorry, will fix that

Reply:
Done